plugins/gcov/gcov.rake

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Based on other ceedling files, when we have `require something` then it is in the first line, in others conditions first line is empty. I noticed this rule from other files, but you can leave it as it is.

Take a look for example here:
https://github.com/ThrowTheSwitch/Ceedling/blob/master/lib/ceedling/tasks_filesystem.rake
https://github.com/ThrowTheSwitch/Ceedling/blob/master/lib/ceedling/task_invoker.rb
https://github.com/ThrowTheSwitch/Ceedling/blob/master/lib/ceedling/stream_wrapper.rb

Reply:
I removed the empty first line in files that begin with `require`. If it's worth commenting about, then it's worth changing.